packages/react/tests/e2e-storybook/cypress/support/commands.js

```diff
@@ -26,3 +26,20 @@ Cypress.Commands.add('mockMastheadFooterData', () => {
     fixture: 'typeahead.json',
   });
 });
+
+/**
+ * Cycle through carbon themes and take a screenshot
+ */
+Cypress.Commands.add('carbonScreenshot', (screenshotOpts = {}) => {
```

Comment:
Awesomeee!!! just small name change
```suggestion
Cypress.Commands.add('carbonThemesScreenshot', (screenshotOpts = {}) => {
```

Reply:
@jeffchew I just removed the ready to merge label for this